sdk/storage/azure-storage-blob-cryptography/src/test/java/com/azure/storage/blob/specialized/cryptography/EncryptedBlockBlobApiTests.java

```diff
@@ -1596,6 +1598,58 @@ public void perCallPolicy() {
             client.getPropertiesWithResponse(null, null, null).getHeaders().getValue("x-ms-version"));
     }
 
+    @ParameterizedTest
+    @ValueSource(strings = {"ENCRYPTIONDATA", "EncryptionData", "eNcRyPtIoNdAtA"})
+    public void encryptionDataCaseInsensitivity(String newKey) {
+        byte[] data = getRandomByteArray(Constants.KB);
+        bec = getEncryptionClient(EncryptionVersion.V2, generateBlobName());
```

Comment:
This test looks really good, can we also test V1?  We may be able to do this by adding another parameter to this test and the test below.

Reply:
Added tests for V1 as well